docs/reference/rest-api/security/get-builtin-privileges.asciidoc

```diff
@@ -78,7 +78,6 @@ A successful call returns an object with "cluster", "index", and "remote_cluster
     "manage_behavioral_analytics",
     "manage_ccr",
     "manage_connector",
-    "manage_data_frame_transforms",
```

Comment:
I don't see mention of removing these privileges in the breaking changes request -- we only mention roles. The `manage_data_frame_transforms` and `monitor_data_frame_transforms` privileges are indeed deprecated according to our docs (`monitor_data_frame_transforms` has already been removed from docs it seems), but we don't issue any deprecation warnings around these privileges (currently there's no clean, built-in support for that). These privileges can be assigned to user-defined roles and user-created API keys so they are not just scoped to the built-in roles we're removing here. 

I think it's likely still a good idea to remove the privileges in 9.0 but I prefer that we do this in a separate PR and after making sure that it's worth the trouble. 

@dan-rubinstein could you revert the changes around privileges in this PR? That'll also take care of some of the CI failures. I can open a follow up PR with the privileges removed -- since it requires a PR against the Serverless repo as well, it'll be easier for me to author both PRs than only one of them.

Reply:
I can revert the changes for this file - to clarify though, would you prefer that we revert the change entirely, or would you prefer that we set the permissions as empty via `Set.of()`?

Reply:
@prwhelan let's revert it entirely -- I synced on this with the ES sec team before the holidays and the preference from our end would be to follow the deprecate, then remove approach, i.e., implement a deprecation warning for the privilege, then remove in a later version. This would delay the removal of the privilege since we'd first need a mechanism for issuing deprecation warnings for privileges (that's on the ES security team to implement). 

Would the following be acceptable for the ML team? Remove the roles now, wait for ES security to implement a deprecation mechanism, deprecate the privilege, then remove it in some later ES version? This way things are less disruptive for any customers that may still be relying on the privilege. 

Just a preference: if you have confidence that only very few (if any) customers still use this privilege, or feel strongly about making it non-functional sooner, I'm also good with making it a NOOP via `Set.of()`. 

Either way though, let's do whatever we do around the privilege in a separate PR to keep things focused.

Reply:
Okay, is there an issue we can track for deprecating and removing the privilege? I imagine we'll need to wait to the v10 upgrade to delete it, in which case I want to record it in some way

Reply:
Yup -- I'll create Jira for deprecating this particular privilege and Slack it to you (it's a private link that external people won't have access to).